config/consumers.py

```diff
@@ -17,27 +25,28 @@ async def connect(self):
 
         # Filter WebSocketSequenceElements by actiontype
         elements = await sync_to_async(list)(
-            WebSocketSequenceElement.objects.filter(action_type__name=self.actiontype).order_by("order_num")
+            WebSocketSequenceElement.objects.filter(action_type__name=self.actiontype).order_by("order_num"),
         )
         if not elements:
-            logging.warning(f"No elements found for actiontype={self.actiontype}.")
+            logger.warning("No elements found for actiontype=%s.", self.actiontype)
             return
 
         # Avoid lazy evaluation
         routes = await sync_to_async(list)(Entry.objects.filter(is_active=True).values_list("route__route", flat=True))
 
         for route in routes:
             for element in elements:
-                msg = await sync_to_async(lambda: element.websocketmessage)()
+                msg = await sync_to_async(partial(element.websocketmessage))
```

Comment:
What does this do?

Reply:
The issue here was this: https://docs.astral.sh/ruff/rules/function-uses-loop-variable/

I'm not sure if this was really an issue, but seemed easy enough to change.

`partial` will essentially prepare a new function for you.